### DISCUSSION TITLE
File upload failed

### DISCUSSION THREAD

**Discussion by grantfwilliams** [requarks/wiki#5298] [Error / Bug Report]
any time I try to upload any image I get the in red: Error during upload / File upload failed.

![image](https://user-images.githubusercontent.com/105661983/168634683-3cb5a454-12ae-48fe-9db2-1a43f0ec5a09.png)

**Comment by danieleteti** [comment]
Same problem here.
Any news about this?

**Comment by NGPixel** [comment]
You'll need to provide a bit more info. Open the browser devtools to see the actual network error.

- Is your wiki behind a reverse proxy?
- Is there any error in the logs?

**Reply by danieleteti** [reply]
No errors anywhere.
I just solved adding permissions on path starting with "/" for write/read assets. I found this in an answer in GitHub, however it's not clearly explained IMO.
However, now it's solved.
Thank you
